### DISCUSSION TITLE
flaresolverr windows exe package and chromium versions

### DISCUSSION THREAD

**Discussion by garfield69** [FlareSolverr/FlareSolverr#757] [General]
Found one caveat of using the windows executable flaresolverr, which is that when your local browser upgrades to chrome 112 the flaresolverr exe remains at 111 causing issues for those jackett indexers that are using the UA in the config fetched from your local browser session.
This is why I prefer using the source github clone with a python flaresolverr.py cmd shell, since I can just restart it and its in sync with my browser version straight away.

My whole point of pointing this out is that whoever schedules the flaresolverr builds will need to remember to build a new release whenever chrome bumps their version, so as to keep current.

**Comment by ngosang** [comment]
When FlareSolverr starts it checks the Chrome/Chromium versions installed, patches the Driver and gets the User-Agent.
* If you use the "compiled package for windows" (flaresolverr_windows_x64.zip) you shouldn't have any issues since Chrome is not going to update itself.
* If you use the Docker image the same thing applies, the browser won't update
* If you are using FlareSolverr from source code with Chrome installed in the system, Chrome will update eventually and FlareSolverr + Jackett stop working. You have to restart FlareSolverr (to patch the new driver and get the new User-Agent) and then restart Jackett to configure the FlareSolverr User-Agent properly.
